Sources/Private/Experimental/Layers/ImageLayer.swift

```diff
@@ -24,12 +24,12 @@ final class ImageLayer: BaseCompositionLayer {
   /// Called by CoreAnimation to create a shadow copy of this layer
   /// More details: https://developer.apple.com/documentation/quartzcore/calayer/1410842-init
   override init(layer: Any) {
-    guard let layer = layer as? Self else {
```

Comment:
Interesting, didn't know this used to be unsupported

Reply:
This was supported, it's the fact that we're using `layer` in the fatalError` that was unsupported

Reply:
oooh I see